tests/cupy_tests/manipulation_tests/test_shape.py

```diff
@@ -176,6 +176,19 @@ def test_external_ravel(self, xp):
         a = a.transpose(2, 0, 1)
         return xp.ravel(a)
 
+    @testing.for_orders('CFA')
+    @testing.numpy_cupy_array_equal()
+    def test_ravel_non_contiguous(self, xp, order):
+        a = xp.array([1])
+        b = xp.broadcast_to(a, (10,))
+        assert not b.flags.c_contiguous and not b.flags.f_contiguous
+        b = b.ravel(order)
+        if order == 'C':
+            assert b.flags.c_contiguous
+        else:
+            assert b.flags.f_contiguous
+        return b
+
```

Comment:
Add the test before `test_external_ravel`. `xp.ravel(a, order='F')` could be tested in future.